src/e2e.js

```diff
@@ -78,6 +111,7 @@ mye2e.prototype.exportKey = function() {
 };
 
 mye2e.prototype.signEncrypt = function(data, encryptKey, sign) {
+  Promise.resolve(refreshBuffer(5000));
```

Comment:
Yeah that would be better, this was just a lazy first attack that got things running. Adding the `.then` at the bottom as in the setup changed the return type, but it should be changeable in another way.